app/src/components/items/viewer/fallback.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+
+import { ItemModel } from "../../../models/item";
+import React from "react";
+import {
+  Heading,
+  Banner,
+  Notification,
+  Box,
+  Link,
+  Icon,
+} from "@nypl/design-system-react-components";
+
+interface ItemProps {
+  item: ItemModel;
+}
+
+const ItemMediaViewerFallback = ({ item }: ItemProps) => {
+  const divisionLink = item?.metadata?.locations?.split("<br>")[0];
```

Comment:
could we add a comment with that assumption, that the first location entry has to be the division link?